src/integration-test/groovy/au/org/ala/fieldcapture/OrganisationProjectSpec.groovy

```diff
@@ -26,4 +26,26 @@ class OrganisationProjectSpec extends StubbedCasSpec {
             projectContent[1].text() == "Project Script Injection <script>alert('Test')</script>"
         }
     }
+
+    void "Can setup organisation configuration in the admin tab"() {
+
+        setup:
+        loginAsMeritAdmin(browser)
+
+        when:
+        to Organisation, orgId
+
+        then:
+        waitFor {at Organisation}
+
+        openAdminTab()
+        adminTabContent.openConfig()
+
+        when:
+        adminTabContent.configOverride.config = '{ "visibility": "public" }'
```

Comment:
What is config override?  I don't think organisations have any original config to override?